src/components/body/body-row.component.ts

```diff
@@ -79,7 +80,7 @@ export class DataTableBodyRowComponent {
 
   @HostBinding('style.width.px')
   get columnsTotalWidths(): string {
-    return this.columnGroupWidths.total;
+    return (this.columnGroupWidths.total - 15) + '';
```

Comment:
@keyeMyria where did 15 come from? If you need a constant value, it's easier to read with a `const veryDescriptiveName = 15;` first